src/platform/testing/e2e/cypress/plugins/index.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs-extra');
 const path = require('path');
+const del = require('del');
```

Comment:
Does `del` have an advantage in this case over something like `fs.unlInkSync`?

Reply:
I'm not sure, I can test it out. I borrowed this from here: https://github.com/cypress-io/cypress/issues/2522#issuecomment-749316813